andela/isildur-rc
# Feature/151207874/effective onboarding of users

#### What does this PR do?
Users should effectively be on-boarded to the app

#### Description of Task to be completed?
Admin/Vendor Users should be provided with a tutorial of how to effectively and seamlessly use the app(reaction commerce) to setup there e-commerce outfit/shop/marketplace.

**Business/User Value**
As an authenticated admin/vendor user, when I navigate to the getting started page, I should be able to follow a guide/tutorial on how  to  setup my e-commerce platform and what each item/buttons in the app does.

**Acceptance Criteria**
GIVEN I successfully signed in or just signed up
WHEN I navigate to the getting started menu
THEN I should be see a getting started page
THEN I should be able to take tutorial on how to use the app to setup my shop/market place

#### How should this be manually tested?
- I should login as an admin/vendor and navigate to the getting started menu
- I should follow the instructions on the getting started page

#### Any background context you want to provide?
Nill

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/151207874

#### Screenshots (if appropriate)
<img width="1280" alt="screen shot 2017-10-08 at 7 34 11 pm" src="https://user-images.githubusercontent.com/12856872/31319889-49b8673e-ac63-11e7-8e2e-72223e5396f6.png">
<img width="1279" alt="screen shot 2017-10-08 at 7 34 23 pm" src="https://user-images.githubusercontent.com/12856872/31319890-52cbfe94-ac63-11e7-956f-3744581c6211.png">
